deployability/modules/testing/tests/test_manager/test_install.py

```diff
@@ -61,7 +61,8 @@ def test_installation(wazuh_params):
 
     # Install managers and perform checkfile testing
     for manager_name, manager_params in wazuh_params['managers'].items():
-        WazuhManager.perform_install_and_scan_for_manager(manager_params, manager_name, wazuh_params)
+        #WazuhManager.perform_install_and_scan_for_manager(manager_params, manager_name, wazuh_params)
```

Comment:
Remove comment

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5324/commits/96c47257f1bca33c7f144285a4c7b63a8c10fbce